workflow_update/starter.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+
+from temporalio.client import Client, WorkflowHandle
+
+from workflow_update import HelloWorldWorkflow
+
+
+class WorkflowRunner:
```

Comment:
I don't think there's much value in this separate class, just put the `start_workflow` in the `run_workflow` method